bin/commands/runs.js

```diff
@@ -69,17 +79,29 @@ module.exports = function run(args) {
       // warn if specFiles cross our limit
       utils.warnSpecLimit(bsConfig, args, specFiles);
 
+      markBlockEnd('preArchiveSteps');
       // Archive the spec files
+      markBlockStart('zip');
+      markBlockStart('zip.archive');
       return archiver.archive(bsConfig.run_settings, config.fileName, args.exclude).then(function (data) {
 
+        markBlockEnd('zip.archive');
         // Uploaded zip file
+        markBlockStart('zip.zipUpload');
         return zipUploader.zipUpload(bsConfig, config.fileName).then(async function (zip) {
+
+          markBlockEnd('zip.zipUpload');
+          markBlockEnd('zip');
           // Create build
 
           //setup Local Testing
+          markBlockStart('localSetup');
           let bs_local = await utils.setupLocalTesting(bsConfig, args);
-
+          markBlockEnd('localSetup');
+          markBlockStart('createBuild');
           return build.createBuild(bsConfig, zip).then(function (data) {
+            markBlockEnd('createBuild');
+            markBlockEnd('total');
```

Comment:
Where have we started the block for total?

Reply:
The internal logic handles this. it'll take from creation of the object

Reply:
Any issues in explicit initialization?

Reply:
no issue as such, it'll be redundant.

Reply:
Then please remove it.